bitnami/redis/templates/sentinel/service.yaml

```
@@ -153,6 +153,9 @@ spec:
       {{- else if eq $masterServiceConfig.type "NodePort" }}
       nodePort:  {{ $redisport }}
       {{- end }}
+      {{- if .Values.sentinel.masterService.extraPorts }}
+      {{- include "common.tplvalues.render" (dict "value" .Values.sentinel.masterService.extraPorts "context" $) | nindent 4 }}
+      {{- end }}
```

I think we should give a try to this, following the same approach as other values in this service:

```suggestion
      {{- if $masterServiceConfig.extraPorts }}
      {{- include "common.tplvalues.render" (dict "value" $masterServiceConfig.extraPorts "context" $) | nindent 4 }}
      {{- end }}
```